src/python/bot/testcase_manager.py

```diff
@@ -821,15 +819,20 @@ def prepare_log_for_upload(symbolized_output, return_code):
   components = revisions.get_component_list(app_revision, job_name)
   component_revisions = (
       revisions.format_revision_list(components, use_html=False) or
-      'Not available.\n')
+      "Not available.\n")
 
-  revisions_header = (
-      'Component revisions (build r{app_revision}):\n{component_revisions}\n'.
-      format(
-          app_revision=app_revision, component_revisions=component_revisions))
-  return_code_header = 'Return code: %s\n\n' % return_code
+  revisions_header =\
+  f'Component revisions (build r{app_revision}):\n{component_revisions}\n'
 
-  result = revisions_header + return_code_header + symbolized_output
+  host_name = environment.get_value('HOSTNAME')
```

Comment:
Probably use bot_name = environment.get_value('BOT_NAME')
i dont know if HOSTNAME is set everywehre.

Reply:
done.